contracts/loan_pool/src/contract.rs

```diff
@@ -165,6 +178,29 @@ impl LoanPoolContract {
         amount
     }
 
+    pub fn add_interest_to_accrual(e: Env) {
+        const DECIMAL: i128 = 10000000;
+        const SECONDS_IN_YEAR: u64 = 31_556_926;
+
+        let current_timestamp = e.ledger().timestamp();
+        let accrual = pool::read_accrual(&e);
+        let accrual_last_update = pool::read_accrual_last_updated(&e);
+        let ledgers_since_update = current_timestamp - accrual_last_update;
```

Comment:
Due to clock synchronization issues, this value could become negative and screw things up.

Should we add `ledgers_since_update >= 0` and handle errors gracefully just to be extra safe?

Unlikely scenario, I know.

Reply:
It's ledger timestamp so unless blockchain runs backwards it won't happen :)